doc/Command-Reference.md

```diff
@@ -48,6 +45,12 @@
 * [ECN](#ecn)
   * [ECN show commands](#ecn-show-commands)
   * [ECN config commands](#ecn-config-commands)
+* [Feature State](#feature-state)
+  * [Feature State show commands](#feature-state-show-commands)
+  * [Feature State config command](#feature-state-config-command)
+* [Feature Auto-restart](#feature-auto-restart)
+  * [Feature Auto-restart show commands](#feature-auto-restart-show-commands)
+  * [Feature Auto-restart config command](#feature-auto-restart-config-command)
```

Comment:
I think we should combine these into one section (i.e., one section per top-level group), otherwise this table of contents will get extremely long. E.g.,

```
* [Feature](#feature)
  * [Feature show commands](#feature-show-commands)
  * [Feature config commands](#feature-config-commands)
```

Then list all the commands under each group. This is how the other sections are currently laid out.

Reply:
@jleveque updated accordingly